cmd/shortener/main.go

```diff
@@ -1,21 +1,88 @@
 package main
 
 import (
-	"log"
+	"context"
+	"errors"
 	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
 
 	"github.com/patraden/ya-practicum-go-shortly/internal/app/config"
-	"github.com/patraden/ya-practicum-go-shortly/internal/app/handler"
-	"github.com/patraden/ya-practicum-go-shortly/internal/app/service"
+	"github.com/patraden/ya-practicum-go-shortly/internal/app/logger"
+	"github.com/patraden/ya-practicum-go-shortly/internal/app/repository"
+	"github.com/patraden/ya-practicum-go-shortly/internal/app/server"
+	"github.com/patraden/ya-practicum-go-shortly/internal/app/urlgenerator"
+	"github.com/rs/zerolog"
 )
 
 func main() {
-	cfg := config.LoadConfig()
-	service := service.NewShortenerService(cfg.URLGenTimeout)
-	r := handler.NewRouter(service, cfg)
-	err := http.ListenAndServe(cfg.ServerAddr, r)
+	log := logger.NewLogger(zerolog.InfoLevel).GetLogger()
+	config := config.LoadConfig()
+	repo := repository.NewInMemoryURLRepository()
+	gen := urlgenerator.NewRandURLGenerator(config.URLsize)
+	srv := server.NewServer(repo, gen, config)
+	manager := repository.NewStateManager(config, log)
+
+	// Initialize repository with the help of repo state manager.
```

Comment:
[SUGGESTION] You can read data from file and then create a repo. You can provide data to the constructor.